data/json/recipes/weapon/piercing.json

```diff
@@ -953,5 +953,22 @@
     "qualities": [ { "id": "CHISEL", "level": 3 } ],
     "tools": [ [ [ "swage", -1 ] ] ],
     "components": [ [ [ "2x4", 1 ], [ "stick", 1 ] ], [ [ "fur", 2 ], [ "leather", 2 ] ] ]
+  },
+  {
+    "result": "knuckle_skewer",
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_PIERCING",
+    "skill_used": "fabrication",
+    "difficulty": 6,
+    "time": "8h",
```

Comment:
Again, 8 hours and the blacksmithing proficiency to craft "metal spike with bracers"

Reply:
using a pre-existing spike would require an additional sawing tool and welding (forge welding is no easy feat with potentially mismatched alloys). Forging your own spike, which is simple in itself, serves for a more practical recipe. Also dropped the time from 8 hours to 4 hours.